test/functional/sqlsrv/MsHelper.inc

```diff
@@ -475,16 +506,17 @@ function executeQuery($conn, $sql, $conds = null, $values = null)
             $sql = $sql . " WHERE $clause ";
         } elseif (!empty($conds)) {
             $sql = $sql . " WHERE $conds ";
-        }        
-        $stmt = sqlsrv_query($conn, $sql);
+        }   
+
+        $stmt = sqlsrv_query($conn, $sql, null, $options);
     } else {
         // with AE enabled, use sqlsrv_prepare() in case there are 
         // fields with unlimited size
-        if (empty($conds) || empty($values)) {
-            $stmt = sqlsrv_prepare($conn, $sql);
+        if (empty($conds) || empty($values)) {            
+            $stmt = sqlsrv_prepare($conn, $sql, null, $options);
```

Comment:
What happens if conds is empty but values is not empty? Doesn't that mean the parameter's not bound? Also, what if conds is not empty but values is empty, then the WHERE clause cannot be executed.

Reply:
Since it's a helper method for the sqlsrv tests and if conds/values aren't right the affected tests will fail. It's up to the caller to ensure the combinations are correct

Reply:
Should still cover those cases (at least with messages saying the user isn't supposed to do this?). Also, put this into the description of this function so the user knows he/she isn't supposed to call this function with this combination of parameters.

Reply:
I agree, but there's a valid case when `$conds` is not empty but `$values` is null. For example, the condition is `aCol IS NULL` or `aCol IS NOT NULL`

Reply:
Just one last suggestion regarding this. Seems like `$values` is only for placeholders within `$cond`. Can you specify this in the description of the function? e.g., `@param  array  $values : array of parameters for placeholders in $cond, null by default`